### PR TITLE
Initial react setup bd

### LINKED ISSUE #32: Add initial structure for react application
add all component files and testing/css

### PR DESCRIPTION
**Pull request by BlakeDonn**
#### What's this PR do?  

- Adds folders for all components with test / css / component file nested 
- Adds dependancies for various packages needed for testing/routing/react
- Adds `apiCalls` for all activities and genres if applicable 
- Adds initial structure for APP and homepage

#### Where should the reviewer start? 

- Start by running `npm install` to install necessary dependencies 

#### How should this be manually tested?  

- Run `npm start` to load the application in the browser 
- Check for file inconsistencies throughout the application

#### Any background context you want to provide?  

- Still debating how the data / action flow should work for our application  .

#### What are the relevant tickets?  

closes #32

### CODE REVIEW

File: src/Components/App/App.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import './App.scss';
+import {Homepage} from '../HomePage/Homepage'
```
Comment (nicolegooden) [inline src/Components/App/App.js:2]: We should consider making this consist from the get-go.  Do we want `export default` syntax for our components, or just to export them directly upon definition?  I think it's easier to just `export default` but that's just my suggestion.

File: package.json [hunk 2]
```diff
@@ -7,13 +7,16 @@
...
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-player": "^2.6.2",
+    "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.0",
     "web-vitals": "^0.2.4"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "PORT=5000 react-scripts start",
```
Comment (Atos20) [inline package.json:19]: I think the port number was an excellent choice.

File: src/Components/HomePage/Homepage.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import './Homepage'
+
+export function Homepage() {
+    return (
+        <main>
+            <article>HOLA</article>
+            <section>
+                <article>Adult Activities</article>
```
Comment (nicolegooden) [inline src/Components/HomePage/Homepage.js:9]: We're probably going to want to have `heading` tags within each of these articles to display the text.

File: src/apiCalls.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
+export const getAllMovies = async () => {
```
Comment (nicolegooden) [inline src/apiCalls.js:1]: Thanks for getting these taken care of early on!

**Review by nicolegooden — APPROVED** [review]
Thanks for handling the initial set up!

File: src/Components/App/App.js (hunk 1 above)
Comment (Atos20) [reply src/Components/App/App.js:2]: I prefer to export a single element when possible, especially when exporting components meaning something like this `export const Component = () => {}`

File: src/Components/App/App.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+import Header from '../Header/Header'
+import React, {Component} from 'react'
+import {Route} from 'react-router-dom'
+
```
Comment (Atos20) [inline src/Components/App/App.js:6]: For readability, I thin it would be better to add extra spaces within `{}`

File: src/Components/App/App.js [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+class App extends Component {
```
Comment (Atos20) [inline src/Components/App/App.js:7]: good call, on turning App into a class components.

File: src/apiCalls.js [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+  }
+}
+export const getPodcastsByGenre = async (genre) => {
+  const response = await fetch(`localhost:3000/api/v1/podcasts/${genre}`)
+  if (response.ok) {
+    return await response.json()
+  } else {
+    return response.error
+  }
+}
```
Comment (Atos20) [inline src/apiCalls.js:64]: Thanks for writing the `if` `else` statements

**Review by Atos20 — APPROVED** [review]

**Pull request merged**